src/test/java/org/springframework/data/couchbase/core/mapping/BasicCouchbasePersistentPropertyTests.java

```diff
@@ -82,6 +83,93 @@ void testSdkIdAnnotationEvaluatedAfterSpringIdAnnotationIsIgnored() {
 		assertThat(test.getIdProperty()).isEqualTo(springIdProperty);
 	}
 
+	@Test
+	void testIdFieldOnly() {
+		{ // only has @springId
```

Comment:
can you split these up into individual unit tests?

Reply:
Done